inlong-agent/agent-installer/conf/installer.properties

```diff
@@ -0,0 +1,44 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#######################
+#   common config
+#######################
+local.ip=127.0.0.1
+agent.enable.oom.exit=false
+
+############################
+# manager config
+############################
+manager.addr=http://127.0.0.1:8083
+manager.auth.secretId=
+manager.auth.secretKey=
+
+############################
+# cluster config for automatically report and register
+############################
+cluster.tag=default_cluster
+cluster.name=default_agent
+inCharges=admin
```

Comment:
why do we need `inCharges`? It's a fixed value or could be changed.

Reply:
no need